android/res/layout/dialog_default_update.xml

```diff
@@ -17,56 +17,63 @@
     android:layout_height="wrap_content"
+    android:background="@drawable/default_dialog_background"
     android:orientation="vertical"
-    android:scrollbars="vertical"
-    android:background="@drawable/default_dialog_background">
+    android:scrollbars="vertical">
 
     <TextView
         android:id="@+id/dialog_default_update_title"
         style="@style/DefaultDialogTheme.TitleBar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content" />
 
-    <TextView
-        android:id="@+id/dialog_default_update_text"
-        style="@style/DefaultDialogTheme.MainText"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:paddingLeft="15dp"
-        android:paddingTop="10dp"
-        android:paddingRight="15dp"
-        android:paddingBottom="10dp" />
 
     <ListView
+        android:id="@+id/dialog_default_update_list_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:id="@+id/dialog_default_update_list_view"></ListView>
+        android:layout_below="@+id/dialog_default_update_text"></ListView>
+
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:background="@color/basic_gray_medium"
         android:orientation="horizontal"
         android:paddingBottom="1dp"
         android:paddingTop="1dp"
-        android:weightSum="1.0"
-        android:background="@color/basic_gray_medium">
+        android:weightSum="1.0">
 
         <Button
             android:id="@+id/dialog_default_update_button_no"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_weight="0.5"
-            android:background="@color/app_light_background"/>
+            android:background="@color/app_light_background"
+            android:visibility="visible" />
 
         <Button
             android:id="@+id/dialog_default_update_button_yes"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginLeft="1dp"
             android:layout_weight="0.5"
-            android:background="@color/app_light_background"/>
+            android:background="@color/app_light_background" />
     </LinearLayout>
-</LinearLayout>
\ No newline at end of file
+
+    <TextView
+        android:id="@+id/dialog_default_update_text"
+        style="@style/DefaultDialogTheme.MainText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/dialog_default_update_title"
+        android:paddingBottom="10dp"
+        android:paddingLeft="15dp"
+        android:paddingRight="15dp"
+        android:paddingTop="10dp" />
+</RelativeLayout>
```

Comment:
I'll never understand why these relative layouts have to add everything backwards